presto-hive-hadoop2/bin/run_hive_tests.sh

```diff
@@ -8,7 +8,7 @@ cleanup_docker_containers
 start_docker_containers
 
 # generate test data
-exec_in_hadoop_master_container su hive -s /usr/bin/hive -f /files/sql/create-test.sql
+exec_in_hadoop_master_container su hive -c '/usr/bin/hive -f /files/sql/create-test.sql'
```

Comment:
I'd prefer to use `sudo` instead, it's `sudo -u hive command args...` syntax is simply better.

Reply:
I would too (and the first thing I tried), but it doesn't exist in the image.

Reply:
https://github.com/prestosql/docker-images/issues/2